### PR TITLE
[IMP] payment_providers/stripe: structure + Apple Pay

### PR DESCRIPTION
**Pull request by auva-odoo**
task-3697828

### CODE REVIEW

File: content/applications/finance/payment_providers/stripe.rst [hunk 1]
```diff
@@ -2,132 +2,135 @@
...
-
-      When you click on **Add endpoint**, your Webhook is configured. You can then click on
-      **reveal** to display your signing secret.
+      #. Go to the `Webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into
+         your Stripe dashboard and go to :menuselection:`Developers --> Webhooks`.
+      #. In the :guilabel:`Hosted endpoints` section, click :guilabel:`Add endpoint`. Then, in the
+         :guilabel:`Endpoint URL` field, enter your Odoo database's URL, followed by
+         `/payment/stripe/webhook`, e.g., `https://yourcompany.odoo.com/payment/stripe/webhook`.
+      #.  Click :guilabel:`Select events` at the bottom of the form, then, in the
+          :guilabel:`Checkout` section, select :guilabel:`checkout.session.completed`, then click
```
Comment (AntoineVDV) [inline content/applications/finance/payment_providers/stripe.rst:92]: Since Odoo 16, there are actually more events than `checkout.session.completed` to select. The full list is hardcoded [here](https://github.com/odoo/odoo/blob/16.0/addons/payment_stripe/const.py#L38-L45) under the `HANDLED_WEBHOOK_EVENTS` constant. When forward-porting this PR, you'll need to monitor `HANDLED_WEBHOOK_EVENTS` over versions (both SaaS and major releases; use the dropdown on the left) and accordingly update the list of events in the documentation.

File: content/applications/finance/payment_providers/stripe.rst [hunk 2]
```diff
@@ -2,132 +2,135 @@
...
 
-To enable some of these local payment methods with Stripe, list them as supported payment icons. To
-do so, go to :menuselection:`Payment Providers --> Stripe --> Configuration` and add the desired
-payment methods in the **Supported Payment Icons** field. If the desired payment method is already
-listed, you do not have anything to do.
-
-.. image:: stripe/payments-config.png
-   :alt: Select and add icons of the payment methods you want to enable
+To adapt the list of enabled payment methods, go to the :guilabel:`Configuration` tab and edit the
+list of :guilabel:`Supported Payment Icons`.
```
Comment (AntoineVDV) [inline content/applications/finance/payment_providers/stripe.rst:121]: Pay attention that "Payment Icons" were renamed to "Payment Method" in 16.2.

**Review by AntoineVDV — APPROVED** [review]
LGTM!

File: content/applications/finance/payment_providers/stripe.rst [hunk 3]
```diff
@@ -2,132 +2,135 @@
...
    - If a payment icon record does not exist in the database and its related local payment method is
      listed above, it is considered enabled with Stripe.
    - If a local payment method is not listed above, it is not supported and cannot be enabled.
 
+Enable Apple Pay
+================
+
+To :ref:`allow customers to use the Apple Pay button <payment_providers/express_checkout>` to pay
+their eCommerce orders, go to the :guilabel:`Configuration` tab, enable :guilabel:`Allow Express
+Checkout`, and click :guilabel:`Enable Apple Pay`.
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:133]: Is Google Pay also accepted? If so, I think this header should be changed to reflect that ("Enable Express Checkout" or "Enable Apple Pay and Google Pay") and the rest of the text should include Google Pay as well.

File: content/applications/finance/payment_providers/stripe.rst [hunk 4]
```diff
@@ -2,132 +2,135 @@
...
 businesses to accept **credit cards** and other payment methods.
 
-Link your Stripe Account with Odoo
-==================================
+Create your Stripe account with Odoo
+====================================
 
 .. seealso::
-   - :ref:`payment_providers/add_new`
+   :ref:`payment_providers/add_new`
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:12]: I would also link to the PoS payment provider page for Stripe: https://www.odoo.com/documentation/master/applications/finance/payment_providers.html
I understand that this page is for online payments, but someone looking for information on a Stripe PoS terminal could easily end up on the wrong page if they just search "Stripe" and click on the wrong "Stripe" page.

Also, is there a reason this seealso is within the "Create your Stripe account with Odoo" header? I think it would be better before the header unless you have a reason you prefer it here :)

File: content/applications/finance/payment_providers/stripe.rst [hunk 5]
```diff
@@ -2,132 +2,135 @@
 Stripe
 ======
 
-`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider, allowing
+`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider allowing
 businesses to accept **credit cards** and other payment methods.
 
-Link your Stripe Account with Odoo
-==================================
+Create your Stripe account with Odoo
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:8]: If a user already has a Stripe account and they just want to _connect_ their existing Stripe account with Odoo, do they follow the same process? If so, I think it would be worth including that ("Create or connect your Stripe account with Odoo"). If not, another section that explains how to connect an existing account would be valuable.

File: content/applications/finance/payment_providers/stripe.rst [hunk 6]
```diff
@@ -2,132 +2,135 @@
...
+         You can select other events, but they are currently not processed by Odoo.
 
 .. _stripe/local-payment-methods:
 
 Enable local payment methods
 ============================
 
-Local payment methods are payment methods that are only available for certain merchants and
-customers countries and currencies.
+Local payment methods are payment methods that are only available for specific merchants and
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:109]:
```suggestion
Local payment methods are payment methods that are only available for specific merchants' and
```
I'm not positive about this apostrophe - it depends on if local payment methods are only available for specific merchants or for merchants of specific countries and currencies.

File: content/applications/finance/payment_providers/stripe.rst [hunk 7]
```diff
@@ -2,132 +2,135 @@
...
-listed, you do not have anything to do.
-
-.. image:: stripe/payments-config.png
-   :alt: Select and add icons of the payment methods you want to enable
+To adapt the list of enabled payment methods, go to the :guilabel:`Configuration` tab and edit the
+list of :guilabel:`Supported Payment Icons`.
 
 .. note::
    - If a payment icon record does not exist in the database and its related local payment method is
      listed above, it is considered enabled with Stripe.
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:125]:
```suggestion
     listed above, it is automatically enabled with Stripe.
```
I think this is more clear, but it's up to you :)

**Review by dade-odoo — APPROVED** [review]
A few minor notes :) we can discuss in person if you like

File: content/applications/finance/payment_providers/stripe.rst [hunk 8]
```diff
@@ -2,132 +2,132 @@
...
-   *Connect Stripe* button.
-
-.. important::
-   If you are testing Stripe (in **test mode**), change the **State** to *Test Mode*. We recommend
-   doing this on a test Odoo database rather than on your main database.
+   - To use an existing Stripe account, :ref:`activate the Developer mode <developer-mode>` and
+     :ref:`enable Stripe manually <payment_providers/add_new>`. You can then :ref:`Fill in your
+     credentials <stripe/api_keys>`, :ref:`generate a webhook <stripe/webhook>`, and enable the
+     payment provider.
+   - You can also try Stripe as a test using the :ref:`payment_providers/test-mode`. To do so,
```
Comment (dade-odoo) [inline content/applications/finance/payment_providers/stripe.rst:43]:
```suggestion
   - You can also test Stripe using the :ref:`payment_providers/test-mode`. To do so,
```
"You can also try Stripe as a test using the test mode" is a bit repetitive :) I think this is clearer.

**Review by dade-odoo — APPROVED** [review]
Just one spot that I would reword :)

**Comment by auva-odoo** [comment]
@robodoo r+
